functions.js

```diff
@@ -96,7 +100,7 @@ Output:
 */
 
 export function returnAsAString(firstNumber, secondNumber, thirdNumber) {
-    return true;
+    return `${firstNumber}, ${secondNumber}, ${thirdNumber}`;
```

Comment:
The expected output here should be a string of the numbers mushed together. So no commas.